docker/encfs/build.sh

```diff
@@ -8,23 +8,7 @@ pushd $(dirname $0)
 
 # This script builds the encrypted filesystem container
 
-mkdir -p bin
-pushd bin
-echo building azmount
-CGO_ENABLED=0 GOOS=linux go build github.com/Microsoft/confidential-sidecar-containers/cmd/azmount
-echo building remotefs
-CGO_ENABLED=0 GOOS=linux go build github.com/Microsoft/confidential-sidecar-containers/cmd/remotefs
-popd 
-
-echo building get-snp-report
-pushd ../../tools/get-snp-report
-make 
-popd
-cp ../../tools/get-snp-report/bin/get-snp-report ./bin
-cp ../../tools/get-snp-report/bin/get-fake-snp-report ./bin
-
-docker build --tag encfs -f Dockerfile.encfs .
+docker build --tag encfs -f Dockerfile.encfs ../..
 
 # clean up
-rm -rf bin
 popd
```

Comment:
Maybe `popd` can also be removed since `pushd` isn't used anymore?

Reply:
pushd is still on line 7...